Update Kenya section

SonjaAM opened the pull request:
Updated Kenya section with Google Doc text, images, and formatting
Added images to img folder

SonjaAM committed:
update text for Kenya section

SonjaAM committed:
upload Kenya images from Google Doc

files are called KEN1.jpg and KEN2.pjg

SonjaAM committed:
Add images KEN1.jpg and KEN1.jpg

SonjaAM committed:
fix KEN2.jpg to KEN2.JPG

SonjaAM committed:
moved image to match Google doc

I am uncertain which section the KEN1.jpg image belongs to (the image of the woman with the black headshawl,purple/blue shirt, and baby on her back and right above Kenya: going to school hungry). I can move and rename if needed.

Zibethin commented:
Hi, thanks a lot for this! Would it be ok to separate each sentence into a new paragraph please? It's a requirement from Comms :) The country name should also be in red (see other titles). Thanks!

SonjaAM commented:
Yep, I'll get right on it.

SonjaAM committed:
change Kenya title to red

added span element to Kenya title to make it red

SonjaAM committed:
move up numbers on the map

Moved the divs containing map numbers up by changing #in-need { top: } from 20% to 17% and changing #food-need { top: and font-size: } from 30% to 25% and 2em to 1.7em

SonjaAM committed:
moved map numbers higher and sub-text smaller

changed #in-need {top: } to 15% and changed ~food-need {top: and font-size: } to 23% and 1.6em

SonjaAM committed:
move #in-need and #food-need

changed #in-need {top: } to 12% and #food-need {top: and font-size: } to 20% and 1.7em

SonjaAM committed:
Merge pull request #1 from SonjaAM/Sonja

Sonja

SonjaAM committed:
try to make/add idp to js

SonjaAM committed:
try adding #i-d-p again

SonjaAM committed:
try to fix idp number

SonjaAM committed:
create idpHtml

SonjaAM committed:
add #i-d-p styling

SonjaAM committed:
added the missed : to #i-d-p styling

SonjaAM committed:
add missed ; in #i-d-p

SonjaAM committed:
Merge pull request #3 from SonjaAM/31Mar-A

added IDP numbers to map

SonjaAM committed:
remove extra <p></p> tags around <ul>

SonjaAM committed:
fixed Kenya headers from h2 to h3 for consistency
